pwa/src/context/queryLimit.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import { GlobalContext } from "./global";
+
+export const queryLimitDefault = 12;

```

Comment:
I'd opt to make this all upper, because its a global exported const `QUERY_LIMIT_DEFAULT`.